Software/app/freertos_lorawan/sys_app.c

```diff
@@ -67,12 +55,7 @@ void SystemApp_Init(void)
   /* Configure the debug mode*/
   DBG_Init();
 
-  /*Initialize the terminal */

```

Comment:
Maybe keep this line (with added space)?

Reply:
Done